src/php/PhelAbstractConfig.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel;
+
+use Gacela\Framework\AbstractConfig;
+use Gacela\Framework\Config;
+use function json_decode;
+
+/**
+ * This is a layer on top of Gacela config, to get the values from the composer.json.
+ */
+abstract class PhelAbstractConfig extends AbstractConfig
+{
+    private array $composerJson = [];
+
+    /**
+     * @override to load the config from the composer.json
+     *
+     * @param null|mixed $default
+     */
+    protected function get(string $key, $default = null)
+    {
+        if (empty($this->composerJson)) {
+            $composerJsonPath = Config::getApplicationRootDir() . DIRECTORY_SEPARATOR . 'composer.json';
+            $this->composerJson = json_decode(file_get_contents($composerJsonPath), true);
+        }
+
+        return $this->composerJson[$key] ?? $default;
```

Comment:
Since everything is in `extra.phel`, we could directly return this.

```php
return $this->composerJson['extra']['phel'][$key] ?? $default
```

Reply:
I like the idea. I will do that 💡